input/parameters/lithium-ion/mcmb2528_lif6-in-ecdmc_lico2_parameters_Dualfoil.csv

```diff
@@ -1,60 +1,59 @@
-Positive electrode cation signed stoichiometry,-1,-,,
-Negative electrode electrons in reaction,1,-,,
-Positive electrode electrons in reaction,1,-,,
-Negative electrode reference exchange-current density,2E-05,Am-2(m3/mol)1.5,Scott Moura FastDFN,Be careful how we implement BV
-Positive electrode reference exchange-current density,6E-07,Am-2(m3/mol)1.5,Scott Moura FastDFN,Be careful how we implement BV
-Reference OCP vs SHE in the negative electrode,,V,,
-Reference OCP vs SHE in the positive electrode,,V,,
-Charge transfer coefficient,0.5,,Scott Moura FastDFN,
-Double-layer capacity,0.2,F.m-2,,
-,,,,
-# Initial Conditions,,,,
-Initial concentration in electrolyte,1000,mol.m-3,Scott Moura FastDFN,
-Initial concentration in negative electrode,19986.609595075,mol.m-3,Scott Moura FastDFN,
-Initial concentration in positive electrode,30730.7554385565,mol.m-3,Scott Moura FastDFN,
+Name [Units],Value,Reference,Notes
+# Empty rows and rows starting with ‘#’ will be ignored,,,
+,,,
+# Physical constants,,,
+Reference temperature [K],298.15,25C,
+,,,
+# Macroscale geometry,,,
+Negative electrode width [m],0.0001,Scott Moura FastDFN,
+Separator width [m],2.5E-05,Scott Moura FastDFN,
+Positive electrode width [m],0.0001,Scott Moura FastDFN,
+Electrode height [m],0.137,KOKAM SLPB78205130H,Not needed for 1D
+Electrode depth [m],0.207,KOKAM SLPB78205130H,Not needed for 1D
+Negative current collector thickness [m],2.5E-05,Scott Moura FastDFN,
+Positive current collector thickness [m],2.5E-05,Scott Moura FastDFN,
+,,,
+# Electrical,,,
+Number of electrodes connected in parallel to make a cell [-],1,-,
+Lower voltage cut-off [V],3.105,-,
+Upper voltage cut-off [V],4.7,-,
+,,,
+# Electrolyte properties,,,
+Typical electrolyte concentration [mol.m-3],1000,Scott Moura FastDFN,
+Cation transference number [-],0.4,Scott Moura FastDFN,
+Typical lithium ion diffusivity [m2.s],5.34E-10,Scott Moura FastDFN,
+,,,
+# Electrode properties,,,
+Negative electrode conductivity [S.m-1],100,Scott Moura FastDFN,graphite
+Positive electrode conductivity [S.m-1],10,Scott Moura FastDFN,lithium cobalt oxide
+Maximum concentration in negative electrode [mol.m-3],24983.2619938437,Scott Moura FastDFN,
+Maximum concentration in positive electrode [mol.m-3],51217.9257309275,Scott Moura FastDFN,
+Negative electrode diffusion coefficient [m2.s],3.9E-14,Scott Moura FastDFN,
+Positive electrode diffusion coefficient [m2.s],1E-13,Scott Moura FastDFN,
+,,,
+# Microstructure,,,
+Negative electrode porosity [-],0.3,Scott Moura FastDFN,
+Separator porosity [-],1,Scott Moura FastDFN,
+Positive electrode porosity [-],0.3,Scott Moura FastDFN,
+Negative particle radius [m],1E-05,Scott Moura FastDFN,
+Positive particle radius [m],1E-05,Scott Moura FastDFN,
+Negative electrode surface area density [m-1],180000,Scott Moura FastDFN,
+Positive electrode surface area density [m-1],150000,Scott Moura FastDFN,
+Bruggeman coefficient [-],1.5,Scott Moura FastDFN,
```

Comment:
Should we actually just leave dimensionless parameters as "Bruggeman coefficient". No need for the brackets. This is consistent with other dimensionless variables